src/status_im2/contexts/chat/messages/navigation/view.cljs

```diff
@@ -51,13 +52,13 @@
                                                           :extrapolateRight "clamp"})]
     [rn/view {:style style/navigation-view}
      [reanimated/blur-view
-      {:blurAmount   32
-       :blurType     (colors/theme-colors :xlight :dark)
-       :overlayColor :transparent
-       :style        (style/animated-blur-view all-loaded? opacity-animation status-bar-height)}]
+      {:blur-amount 16
```

Comment:
I think the top bar blur should be matching with the composer's blur 
`:blur-amount 20`